FrontEndApp/v1-react/src/Components/MainApp/Step2.js

```diff
@@ -2318,15 +2319,24 @@ function Step2() {
 
     setcomponents(tempArr);
 
-
-
-
-
-    
-   
-
-
-  };
+    const recommendations = await getRecommendations(tempArr)
+		console.log(recommendations)
+	}
+
+	const getRecommendations = layersArr => {
+		let recommendationArr = layersArr.map(x => x.name)
+
+		return axios
+			.post("http://localhost:8080/predict", {
```

Comment:
```suggestion
			.post("http://localhost:8080/predict", {
```

Please use env variables.